app/controllers/users_controller.rb

```diff
@@ -1,10 +1,25 @@
 class UsersController < ApplicationController
-  def update_profile
+  def update#update profile
     @current_user.add_image(get_profile_image) if params.keys.include?("user")
-    redirect_to user_profile_path, notice: t('.successfully_uploaded')
+    selected_topics = get_favorite_topics
+    @current_user.add_topics(selected_topics) if selected_topics.present?
```

Comment:
Add topics remove functionality

Reply:
Update profile topics select box should have currently favourite topics selected.